src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -2278,6 +2278,34 @@ public function testTrustedHost()
         $this->assertSame(443, $request->getPort());
     }
 
+    public function testTrustedPrefix()
+    {
+        Request::setTrustedProxies(['1.1.1.1'], Request::HEADER_X_FORWARDED_TRAEFIK);
+
+        //test with index deployed under root
+        $request = Request::create('/method');
+        $request->server->set('REMOTE_ADDR', '1.1.1.1');
+        $request->headers->set('X-Forwarded-Prefix', '/myprefix');
+        $request->headers->set('Forwarded', 'host=localhost:8080');
+
+        $this->assertSame('/myprefix', $request->getBaseUrl());
+        $this->assertSame('/method', $request->getPathInfo());
+
+        //test with index file deployed in subdir, i.e. local dev server (insecure!!)
```

Comment:
I suggest to split the 2 tests in separate test methods rather than bundling them in the same method and hiding that we actually run 2 separate tests.

Reply:
done